plans/062-dev-ui-ingest-same-origin.md

```
@@ -0,0 +1,344 @@
+  export function isLoopbackOrigin(origin: string): boolean {
+    try {
+      const host = new URL(origin).hostname;
+      // WHATWG URL keeps the brackets on IPv6 hostnames ('[::1]'); a bare '::1' never parses.
+      return host === 'localhost' || host === '127.0.0.1' || host === '[::1]';
+    } catch {
+      return false;
+    }
+  }
+  ```
+
+- `packages/vite/src/hooks/handle.ts:42-66` — 送信側 `postIngest`。`fetch(\`${origin}/__svelte-vitals/ingest\`, { method: 'POST', headers: { 'content-type': 'application/json' }, body })`。`origin`は`event.url.origin`(=ページ要求の `Host`由来)で、dashboard と同じ vite サーバーを指す。Node の`fetch`(undici)は `Origin` ヘッダーを付けない。
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)